internal/rbd/nodeserver.go

```diff
@@ -718,8 +725,24 @@ func (ns *NodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpandV
 	}
 
 	diskMounter := &mount.SafeFormatAndMount{Interface: ns.mounter, Exec: utilexec.New()}
-	// TODO check size and return success or error
 	volumePath += "/" + volumeID
+
+	// check size of the mount path
+	var vStat syscall.Statfs_t
```

Comment:
@agarwal-mudit do we still need this change?

for me looks like above volumePath check should be enough to fix resize issue.

Reply:
Yes, this is a precautionary change. Not related to that issue.

Reply:
can we remove this change?

Reply:
I will let @humblec comment on that

Reply:
Removed